### ISSUE TITLE
An input shape error for breast_density_classification

### ISSUE THREAD

**Issue by binliunls** [Project-MONAI/model-zoo#303]
**Describe the bug**
In the config files of breast_density_classificaion, there is a mismatch of input size between `inference.json` and `metadata.json`. In the `inference.json`, the input image is finally resized to size `299`, while the input size in `metadata.json` is `229`. 

Also this model can be converted to a torchscript.

**Comment by yiheng-wang-nv** [comment]
Thanks @binliunls for posting the issue.
As for the repository side, our ci test does not detect the wrong input size issue. I checked and the issue comes from `monai.bundle.verify_net_in_out` side, now I'm working on finding the root cause of it.
As for the bundle side, Hi @vikashg , may need your help to update the bundle to solve the mismatch issue, and as @binliunls shares, your bundle is actually support torchscript. Could you please help to double check it? Thanks!

**Comment by yiheng-wang-nv** [comment]
Update on the size issue:
both `229` and `299` are worked in `network.eval()` mode, which is used by our verify function. Therefore, we did not detect the error.
Hi @vikashg , could you please also add a simple description in the readme and tell users 1) what is the size you used to train the model 2) mention that 229 (or 299) size can only be used in inference. Thanks!

**Comment by vikashg** [comment]
Ok I can do that. I will try to add some details on how to change the image size as well.

**Comment by vikashg** [comment]
Hey @yiheng-wang-nv, I think its already there in the description 
![image](https://user-images.githubusercontent.com/3863212/216968219-b689b8c7-c6b7-4bf9-89ad-ea3c162230cb.png)

**Comment by yiheng-wang-nv** [comment]
> Hey @yiheng-wang-nv, I think its already there in the description ![image](https://user-images.githubusercontent.com/3863212/216968219-b689b8c7-c6b7-4bf9-89ad-ea3c162230cb.png)

Thanks, I mean the size used to train the model, not for inference. It seems 299 cannot be used in training.

**Comment by binliunls** [comment]
> > Hey @yiheng-wang-nv, I think its already there in the description ![image](https://user-images.githubusercontent.com/3863212/216968219-b689b8c7-c6b7-4bf9-89ad-ea3c162230cb.png)
> 
> Thanks, I mean the size used to train the model, not for inference. It seems 299 cannot be used in training.

Hi @yiheng-wang-nv , I think it is the 229 that cannot be used in training.

**Comment by vikashg** [comment]
Ya 229 needs to be used for training as well. I didn't make the training pipeline. @binliunls Do you need a training pipeline as well ?

**Comment by binliunls** [comment]
> Ya 229 needs to be used for training as well. I didn't make the training pipeline. @binliunls Do you need a training pipeline as well ?

No. It just confused me that the input size in the `metadata.json` which is 229 and  the resize size in `inference.json` which is 299  are different.

**Comment by vikashg** [comment]
Ah thanks that must be a mistake thanks for pointing it out. I will correct it.

**Comment by yiheng-wang-nv** [comment]
Hi @binliunls , I tried both 229 and 299 in training mode, and they all failed. The code is:
```
from monai.networks.nets import TorchVisionFCModel
import monai
import torch
import os

model = TorchVisionFCModel(
    model_name="inception_v3",
    num_classes=4,
    use_conv=False,
    bias=True,
    pool=None,
    pretrained=True,
).cuda()

sample = torch.rand(1, 3, 299, 299).cuda()
model(sample)
```

**Comment by binliunls** [comment]
> 

Hi @yiheng-wang-nv ,
      As shown in the [inference config](https://github.com/Project-MONAI/model-zoo/blob/148f387f7ef629ec411389b81a5c03853c189025/models/breast_density_classification/configs/inference.json#L26), the input shape should be (4, 3, 299, 299).

**Comment by vikashg** [comment]
Hi @binliunls Is this resolved ?

**Comment by binliunls** [comment]
> Hi @binliunls Is this resolved ?

Hi @vikashg ,
     I haven't seen any PR to solve this issue. Would you like to make a PR to fix it?

**Comment by vikashg** [comment]
ok I will

**Comment by vikashg** [comment]
@yiheng-wang-nv I am not able to push a commit into the repo. I am getting an authentication error, even though the rsa keys are updated on github.

**Comment by yiheng-wang-nv** [comment]
> inference config

Hi @vikashg , I'm not sure which error you meet, but you may need to fork this repo and then do any changes in a new branch within your fork repo. I just saw you create a branch with this repo directly, it may lead to the error, thus I just closed your PR and delete the branch from this repo.

I discussed with @binliunls , and the bundle may need to change in the follow parts:
1. change 229 into 299 as what you did in the PR.
2. modify "changelog" and "version" since the bundle is changed.
3. There are two "Input and Output Formats" sections in README, and one of them may be an outdated version and should be removed. In addition, could you please also mention what is the input size when you do training? It can help users to avoid size related debug. According to @binliunls and my trials, `(4, 3, 299, 299)` can work when model is in training mode, but `(1, 3, 299, 299)` will raise an error (`(1, 3, 512, 512)` also works.

**Comment by vikashg** [comment]
Thanks @yiheng-wang-nv Sorry I have been unavailable last week. We had some site visits. Thanks fo handling it

**Comment by yiheng-wang-nv** [comment]
Hi @vikashg , could you help to solve the issue? Or maybe I can submit a PR and invite you to review it?

**Comment by vikashg** [comment]
Yes I will do it today for sure. Totally forgot about this issue. 
Thanks for reminding.
